src/messaging/rabbitmq/consumer.ts

```diff
@@ -31,13 +31,19 @@ export default class RMQConsumer {
     try {
       if (!this.isConnected || this.conn === null) {
         this.conn = await this.connect()
+        this.isConnected = true
```

Comment:
This doesn't impact business logic as such. Worked well even when not set to true because the way reconnects were handled. But to have correct state of the variable, setting it to true.